bin/gaffer.cmd

```diff
@@ -161,7 +161,6 @@ if "%GAFFER_DEBUG%" NEQ "" (
 )
 
 if %ERRORLEVEL% NEQ 0 (
```

Comment:
Do we still need this `if %ERRORLEVEL%` stuff now, or does it all boil down to an unconditional `exit /B %ERRORLEVEL%`?

Reply:
Ah yes, that shouldn't be needed now.